packaging/format-docs.py

```diff
@@ -83,7 +83,7 @@ def format_docs(version, collectionName, types):
 
                         print(f'| {prop["PropertyName"]} | {defaultValue} | {prop["UserFriendlyType"]} | {prop["Description"]} |')
                     else:
-                        print(f'| {prop["PropertyName"]} | {prop["DefaultValue"]} | {prop["UserFriendlyType"]} | {prop["Description"]} |')
+                        print(f'| {prop["PropertyName"]} | {prop["DefaultValue"] or ""} | {prop["UserFriendlyType"]} | {prop["Description"]} |')
```

Comment:
Why is this needed? For reverting this change does nothing

Reply:
When you actually have `null` values in the generated JSON, the Python script serializes that as "None", hence this check to not show anything in the Markdown.